Modules/_testcapi/parts.h

```diff
@@ -27,6 +27,7 @@
 int _PyTestCapi_Init_Vectorcall(PyObject *module);
 int _PyTestCapi_Init_Heaptype(PyObject *module);
 int _PyTestCapi_Init_Unicode(PyObject *module);
+int _PyTestCapi_Init_FuncEvents(PyObject *module);
```

Comment:
FYI, I recently split out the dict and type watcher tests into `Modules/_testcapi/watchers.c` and `Lib/test/test_capi/test_watchers.py`. Would you mind redirecting your test code to those files? Sorry for the inconvenience.